mbofui/app/js/factories.js

```diff
@@ -2,6 +2,16 @@
 /* global ocellus, leafletize, _*/
 ocellus.factory('Bof', ['$http', '$log', '$q', '$rootScope', function($http, $log, $q, $rootScope) {
   return {
+    //get user
+    GetUser: function(url) {
+      return $http.get(url).then(
+        function success(result) {
+          return result;
+        },
+        function error(result) {
+          // do some error things
```

Comment:
When are we planning on doing proper error handling? Should a JIRA be written for proper error handling?